frontend/javascripts/dashboard/dataset_view.js

```diff
@@ -184,30 +183,32 @@ function DatasetView(props: Props) {
   );
   const searchBox = (
     <Search
-      style={{ width: 200 }}
-      onPressEnter={handleSearch}
-      onChange={handleSearch}
-      value={searchQuery}
+    style={{ width: 200 }}
+    onPressEnter={handleSearch}
+    onChange={handleSearch}
+    value={searchQuery}
     />
-  );
-  const search = isUserAdmin ? (
-    <InputGroup compact>
-      {searchBox}
-      <Dropdown overlay={filterMenu} trigger={["click"]}>
-        <Button>
-          <Badge dot={datasetFilteringMode !== "showAllDatasets"}>
-            <Icon type="setting" />
-          </Badge>
-        </Button>
-      </Dropdown>
-    </InputGroup>
-  ) : (
-    searchBox
-  );
+    );
+    
+    const isUserAdminOrDatasetManager = Utils.isUserAdmin(props.user) || Utils.isUserDatasetManager(props.user);;
```

Comment:
@hotzenklotz It looks like you don't have prettier enabled, some of the indentations changed and there is a double semicolon here. Prettier should take care of this automatically, alternatively run `yarn pretty` before committing :)

Reply:
> @MichaelBuessemeyer @fm3 Any thoughts on this: Right now only Admins can see all users/teams with access to a dataset in the `DatasetAccessListView`? Should Dataset Managers see those too? Probably, right?
> 
> That raises the issue again, the dataset manager may be able to see too many user names, emails etc. (Or maybe we have to remove/refactor the `DatasetAccessListView` to only show teams, which is fine.)
> 
> https://github.com/scalableminds/webknossos/blob/94c944bc4122763d2e6df3370128b2180cf8d174/frontend/javascripts/dashboard/advanced_dataset/dataset_table.js#L157

According to @fm3 this view should be:
- viewable by admins (see all users with access)
- viewable by team managers (see only users within their teams)
- viewable by dataset managers (see only users within their teams)

Reply:
(meaning that being dataset manager does not help here, only team managers or admins can see content there, so it should be hidden to anyone who is not admin or team manager)

Reply:
So
https://github.com/scalableminds/webknossos/blob/94c944bc4122763d2e6df3370128b2180cf8d174/frontend/javascripts/dashboard/advanced_dataset/dataset_table.js#L157

should be changed to 
```
isUserAdminOrTeamManager ? dataset => <DatasetAccessListView dataset={datset} /> : null
```
?
If that's the case, I would only show the DatasetAccessListView to TeamManagers if they manage a team of the specific dataset, right?